### DISCUSSION TITLE
Welcome to Talk-About-CSS Discussions!

### DISCUSSION THREAD

**Discussion by airen** [airen/Talk-About-CSS#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
What you think is the best and efficient way to center a div, I have been using the following code to that.

```Css
div {
  display:grid;
  place-items:center;
  heights:100vh;
}
```
